src/components/markdown-renderer/replace-components/quote-options/quote-options-replacer.tsx

```diff
@@ -37,6 +37,6 @@ export class QuoteOptionsReplacer implements ComponentReplacer {
       return
     }
     node.attribs = Object.assign(node.attribs || {}, { style: `border-left-color: ${attributes['data-color']};` })
-    return subNodeConverter(node, index)
+    return undefined
```

Comment:
don't we need that?

Reply:
No. A prior version had a bug, where the execution of the sub node converter started a loop and a stack overflow. I fixed this.